org.eclipse.sprotty/src/main/java/org/eclipse/sprotty/SModel.xtend

```diff
@@ -41,6 +42,12 @@ abstract class SModelElement {
 @Accessors
 @ToString(skipNulls = true)
 class SModelRoot extends SModelElement {
+	
+	public static final SModelRoot EMPTY_ROOT = new SModelRoot [
```

Comment:
This constant is mutable. If we offer such a constant, we should override all the setters with UnsupportedOperationException.